server/routes/Article.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from "express";
+import { favoriteOrUnFavoriteArticle } from "../controllers/favorite";
+import { verifyToken } from "../middlewares/authentications/verifyToken";
+import { findArticle } from "../middlewares/articles/findArticle";
+
+const articleRoutes = Router();
+
+articleRoutes.post(
+  "/:slug/favorite",
+  verifyToken,
+  findArticle,
+  favoriteOrUnFavoriteArticle
+);
+
+export default articleRoutes;
```

Comment:
Please rename this file from `Article.js` to `article.js`. This file does not contain a class.
Also, the route should read `/articles/:slug/favorite`. This is because in the validation middleware that we are using, it needs to get the path and then map it to the `req.body` to be able to validate values sent. If we don't include the `articles` in the routing here, the path will read `/` instead of `/articles` and the validation will fail.